pages/Messages/Identifiers.md

```diff
@@ -0,0 +1,91 @@
+---
+name: Identifiers
+route: /Messages/Identifiers
+menu: Messages
+---
+
+# Messages Identifiers
+
+## Specification Status
+
+| Version | Status |
+---------- | ---------
+| 0.1     | Tentative |
+
+## Purpose
+
+1. Describe a means of uniquely identifying DSNP content messages
+1. Describe potential extensions of identifiers for the future
+
+## Assumptions
+
+* All assumptions from [DSNP Messages](/Messages/Overview)
+
+## Identifiers
+
+DSNP message identifiers at minimum consist of a string with three parts, a protocol, a user identifier and a content hash, like so:
+
+```
+dsnp://0123/456789abcdef0123
+```
+
+In this example, `dsnp://` is the protocol indicating that the latest version of this specification should be used.
+`0123` is the identifier representing a user as described in the [Identity Registry](/Identity/Registry) specification.
+`456789abcdef0123` is the hash of the content as described in the [Messages Serialization](/Messages/Serialization) specification.
+
+### Protocol
+
+The protocol section of an identifier can be used to indicate which version of the DSNP specification should be used to identify content.
+If present, any number after the string `dsnp` should indicate the specific version.
+For example, `dsnp1.2.1://` would indicate that version 1, subversion 2, patch release 1 of the specification should be used.
+
+If the patch release or subversion number is omitted, the latest available subversion and patch release should be used.
+For example, `dsnp2://` would indicate the most up to date subversion and patch release under version 2 of the DSNP should be used.
+
+If no version number is provided at all, i.e. `dsnp://`, then the most recent available version of the spec should be used.
+
+### User Identifier
+
+The user identifier section of the identifier string should adhere to the relevant version of the [Identity Registry](/Identity/Registry) specification.
+Generally, this will be a hexadecimal representation of the user's 64 bit registry identifier number.
+For example, `2B02` would indicate user number 690.
```

Comment:
I really wanted to throw in a Kurt Vonnegut reference here, but unfortunately, [2BR02B](https://en.wikipedia.org/wiki/2_B_R_0_2_B) is more than 64 bits in hex.